apps/server/src/mindmap/repository/mindmap.repository.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import Mindmap from '../schema/mindmap.schema';
+import { Model, Types } from 'mongoose';
+
+@Injectable()
+export class MindmapRepository {
+  constructor(
+    @InjectModel(Mindmap.name)
+    private readonly model: Model<Mindmap>,
+  ) {}
+
+  async create(mindmap: Mindmap): Promise<Mindmap> {
+    const doc = await this.model.create(mindmap);
```

Comment:
@BULMA-OO there's no point storing return value in a variable and then returning the variable.